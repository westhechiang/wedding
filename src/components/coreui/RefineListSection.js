import styled from 'styled-components';
import { theme } from '../../ui';

const RefineListSection = styled.div`
  margin-top: 2px;
  padding: 10px 20px;
  background-color: ${theme.color.grayDark};
  width: 100%;
  display: block;
  box-sizing: border-box;
  flex-direction: column;
  border: 1px solid ${theme.color.gray};
`;

export default RefineListSection;
