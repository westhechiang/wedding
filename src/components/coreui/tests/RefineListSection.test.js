import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import RefineListSection from '../../../components/coreui/RefineListSection';

describe('<RefineListSection />', () => {
  it('should match its empty snapshot', () => {
    const wrapper = shallow(<RefineListSection />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
