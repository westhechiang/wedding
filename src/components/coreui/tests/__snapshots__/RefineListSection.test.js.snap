// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RefineListSection /> should match its empty snapshot 1`] = `
<div
  className="sc-bdVaJa jfMYRl"
/>
`;
